src/client.ts

```diff
@@ -77,8 +77,8 @@ export class C8Client extends Fabric {
     dcName: string,
     subscriptionName: string
   ) {
-    const dcList = await this.getDcList();
-    let dcUrl = dcList[0].dcInfo[0].tags.url;
+    const dcList = await this.getLocalDc();
```

Comment:
change the obj name to more relavant name

Reply:
Done

Reply:
Done